### ISSUE TITLE
Can you add SimpleClans support? 

### ISSUE THREAD

**Issue by TomLewis** [Klyser8/Karma#2]
Latest version is on GitHub https://github.com/marcelo-mason/SimpleClans
He has an API.

There is a whole weight/point KDR system in SimpleClans for recording if you kill an Civilian/Neutral/Rival based on your Ally/Rival status with other crews.

Civilian = Person not in a crew
Neutral = Person in a crew but not allied/rivalled
Rival = Person in a rival crew

SimpleClans does all the hard work with setting the weights and recording the data for all this and managing the crews, would be brilliant to get this information into a karma plugin.

**Comment by Klyser8** [comment]
So if I understand this correctly, you want Karma points to be gained/lost depending on the status of the player (Whether he is a Civilian, Neutral or Rival)?

**Comment by TomLewis** [comment]
Yeah, its based on Kill tho, so when a kill is triggered by SimpleClans its recorded as Civilian, Neutral or Rival based on the users crew and who they are rivalled with.

**Comment by Klyser8** [comment]
That sounds simple to add, I will add it on the to-do list for the next update, which I predict will come out by the end of the month.
